README.md

```diff
@@ -1,2 +1,100 @@
-# yearbook
-Digital platform to capture and celebrate the journey of Obsidi Academy's Cohort 6 members 
+# Yearbook
+
+Digital Yearbook for Obsidi Alumni
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Introduction
+
+The Yearbook project is a digital yearbook application for Obsidi Alumni. It is built using Spring Boot for the backend and a frontend framework (to be specified).
+
+## Features
+
+- User authentication and authorization
+- Profile management
+- Yearbook entries with text and images
+- Search functionality
+- Admin panel for managing users and entries
+
+## Installation
+
+### Prerequisites
+
+- Java 21 or higher
```

Comment:
I would leave this as just Java 21 since it is the latest version with LTS. In previous group projects, some people with Java 23 install could not access certain packages from Spring and their code would not compile as written with Java 21 syntax.